client/src/pages/SpendingList.js

```diff
@@ -108,7 +110,9 @@ export default function SpendingList(props) {
         <Background>
           <PageDestination label={journey.title} />
           <HeadingContainer>
-            <PageHeading label={journey.budget} />
+            <PageHeading
+              label={getAvailableBudget(sumOfSpendings(spendingItems), journey.budget) + ' €'}
```

Comment:
Is every spending in euro? Here hard coded ...

Reply:
For this moment and to reach the MVP every spending is in euro. For the future/ after the event a currency changer could be a nice feature to add :)